src/openzaak/management/commands/send_test_notification.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from vng_api_common.notifications.models import NotificationsConfig
+
+
+class Command(BaseCommand):
+    help = "Send a test notification to verify if notifications are properly configured"
+
+    def handle(self, *args, **options):
+        nrc_client = NotificationsConfig.get_client()
+
+        data = {
+            "kanaal": "zaken",
```

Comment:
@sergei-maertens not sure if there is a way to create a `test` kanaal without having do define it in kanalen.py?

Reply:
instantiating a `Kanaal` object should register it :)